tools/opt/opt.cpp

```diff
@@ -89,6 +98,21 @@ int main(int argc, char** argv) {
         }
       } else if (0 == strcmp(cur_arg, "--strip-debug")) {
         pass_manager.AddPass<opt::StripDebugInfoPass>();
+      } else if (0 == strcmp(cur_arg, "--set-spec-const-default-value")) {
+        std::ostringstream spec_ids_vals_stream;
+        // All the arguments with ':' inside will be considered as
```

Comment:
I personally don't think this is a good way to distinguish arguments. We may have additional options in the future that takes `:` somehow, like `--some-future-option=key:value`, which will be considered as `<spec-id>:<default-value>` pair here.

What about the format like `--set-spec-const-default-value="k1:v1 k2:v2"`? The quotation should act as delimiters for us. That's also the typical way of passing values containing spaces. It's also much easier to extract the whole string here.